### PR TITLE
Adding spatial data and associated code

### PR DESCRIPTION
**Pull request by wcornwell**

### CODE REVIEW

File: R/gbif_cleaning_katana.R [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    aus_filt,
+    species,
+    infraspecificepithet,
+    taxonrank,
+    decimalLongitude = decimallongitude,
+    decimalLatitude = decimallatitude,
+    scientificname,
+    verbatimscientificname
+) %>%
+    write_csv("filtered_aus_obs_28feb2023_limited_cols.csv")
```
Comment (dfalster) [inline R/gbif_cleaning_katana.R:153]: How does this name relate to the csv files below? Are they meant to be the same thing?

File: R/gbif_process.R [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+library(tidyverse)
+library(data.table)
+
+sn <- fread("data/gbif/gbif_species_names_0006942-230224095556074.csv")
```
Comment (dfalster) [inline R/gbif_process.R:5]: What is this script achieving? What are the two csv files?

**Review by dfalster — CHANGES_REQUESTED** [review]
Nice work @wcornwell 

What are the two csv files? Is this likely to change?

File: R/gbif_process.R (hunk 2 above)
Comment (wcornwell) [reply R/gbif_process.R:5]: those are the gbif species names for work on taxonomic matching. it's small enough to commit.

File: R/gbif_cleaning_katana.R (hunk 1 above)
Comment (wcornwell) [reply R/gbif_cleaning_katana.R:153]: no that's the giant observation file.  it's now here: https://www.dropbox.com/s/byqor7tke9b19iy/filtered_aus_obs_28feb2023_limited_cols.csv?dl=0  .  the names still have to be synced though, which thomas is working on.

**Review by dfalster — APPROVED** [review]

**Pull request merged**